src/Employer/Employer.Web/Controllers/VacancyViewController.cs

```diff
@@ -20,7 +20,7 @@ public VacancyViewController(VacancyViewOrchestrator orchestrator)
         }
 
         [HttpGet("", Name = RouteNames.DisplayVacancy_Get)]
-        public async Task<IActionResult> DisplayVacancy(VacancyRouteModel vrm)
+        public async Task<IActionResult> DisplayVacancy(VacancyRouteModel vrm, [FromQuery] string showApplications)
```

Comment:
Unsure about this faux boolean query string parameter. Might need to speak with @mgwilliam about this one.

Reply:
@mgwilliam What's your opinion on this? ta

Reply:
@chrisjensenuk i don't have a preference other than we should be consistent with any other similar instances in the code

Reply:
I had a conversation with Chris on this, the boolean is used to decide between two views on the same end point, perhaps we could just have two end points instead.

Reply:
We have precedent for using pseudo booleans as request parameters.  I think adding another 'view vacancy' endpoint just complicates things.  I'm happy as is it.

Reply:
@chrisjensenuk @dev4valtech bear in mind that the dashboard isn't having tabs on for public beta as sorcha has removed them